pandas/tests/tslibs/test_tslib.py

```diff
@@ -0,0 +1,169 @@
+# -*- coding: utf-8 -*-
+import calendar
+from datetime import datetime, date
+
+import numpy as np
+import pytest
+
+from pandas import compat
+from pandas._libs import tslib
+from pandas.compat.numpy import np_array_datetime64_compat
+import pandas.util.testing as tm
+
+
+def test_monthrange():
+    for y in range(2000, 2013):
+        for m in range(1, 13):
+            assert tslib.monthrange(y, m) == calendar.monthrange(y, m)
+
+
+def test_normalize_datetime():
+    actual = tslib.normalize_date(datetime(2007, 10, 1, 1, 12, 5, 10))
+    assert actual == datetime(2007, 10, 1)
+
+
+def test_normalize_date():
+    value = date(2012, 9, 7)
+
+    result = tslib.normalize_date(value)
+    assert (result == datetime(2012, 9, 7))
+
+    value = datetime(2012, 9, 7, 12)
+
+    result = tslib.normalize_date(value)
+    assert (result == datetime(2012, 9, 7))
+
+
+class TestParseISO8601(object):
```

Comment:
these are conversion tests no? (so separate file). I don't really want much in tslib/test_testlib.py except if it doesnt' fit anywhere else